web/modules/custom/hello_user/hello_user.permissions.yml

```diff
@@ -0,0 +1,7 @@
+# This key name of the permission will be used further in code.
+hello user permission:
```

Comment:
Where is it used?

Reply:
I have added this permission in the hello_user.permissions.yml. Do I need to check dynamically in php code?